lametro/models.py

```diff
@@ -41,6 +41,20 @@ def web_source(self):
     def api_source(self):
         return self.sources.get(note='api')
 
+    @property
+    def api_representation(self):
+        response = requests.get(self.api_source.url)
+
+        if response.status_code != 200:
+            msg = 'Request to {0} resulted in non-200 status code: {1}'.format(
+                self.api_source.url, response.status_code
+            )
+            logger.warning(msg)
+            return None
+
+        else:
+            return response.json()
```

Comment:
Migrated from views.py. Now, we can get at the API representation of any board report, organization, person, or event through the Django shell!